BackEnd/profiles/views.py

```diff
@@ -264,3 +270,28 @@ class RegionDetail(RetrieveUpdateDestroyAPIView):
     serializer_class = RegionSerializer
     permission_classes = (IsAdminUser,)
     queryset = Region.objects.all()
+
+
+class ProfileModeration(APIView):
```

Comment:
Use `generics` whereever possible (and to your best to follow RESTful approach).